src/fsc/fscProject/fsc.fsproj

```diff
@@ -11,12 +11,14 @@
   <PropertyGroup Condition="'$(Configuration)' == 'Proto'">
     <TargetFramework>$(FSharpNetCoreProductTargetFramework)</TargetFramework>
     <PublishReadyToRun>true</PublishReadyToRun>
+    <PublishReadyToRunCrossgen2ExtraArgs>--embed-pgo-data</PublishReadyToRunCrossgen2ExtraArgs>
     <RuntimeIdentifier>$(NETCoreSdkRuntimeIdentifier)</RuntimeIdentifier>
   </PropertyGroup>
 
   <ItemGroup Condition="'$(Configuration)' == 'Proto'">
     <PublishReadyToRunExclude Include="FSharp.Build.dll" />
     <PublishReadyToRunExclude Include="FSharp.Core.dll" />
+    <PublishReadyToRunPgoFiles Include="$(MibcPath)" />
   </ItemGroup>
```

Comment:
This will only enable it in proto, it needs to be enabled in product as well

Reply:
Yeah that's fixed I guess.